.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
```

Comment:
Still reading it, but I didn't notice any grunt-specific stuff in this PR. Is the expectation that this would be absorbed in a higher kind of grunt workflow?

We should be using gulp at edX, since we are already using that on a couple of repositories. But, seems like this is more about the ignoring, so I was curious.